### PR TITLE
Front end changes

### PR DESCRIPTION
**Pull request by EvanBarbour3**
## Features
- Added logo and changed favicon
- Changed tabs so they look less 'bulky' and changed the format of the forms to match this
- Changed glyphicons and added 'scripts' section to master

[Delivers#100803884]

### CODE REVIEW

File: public/ain-logo-standard-medium.svg [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (wnowicki) [inline public/ain-logo-standard-medium.svg]: should it go directly to `/public` ??

if possible please create some folder for images

File: resources/views/includes/form/json.blade.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            }
+            else {
+                $rtn .= '<dd>' . $v .'</dd>';
+            }
+        }
+        $rtn .= '</dl>';
+        return $rtn;
+    }
+
+?>
```
Comment (wnowicki) [inline resources/views/includes/form/json.blade.php]: CS

Comment (EvanBarbour3) [inline resources/views/includes/form/json.blade.php]: Done

File: public/ain-logo-standard-medium.svg (hunk 1 above)
Comment (EvanBarbour3) [inline public/ain-logo-standard-medium.svg]: Image directory has been created in public, known used images have been moved

**Pull request merged**